src/fu-engine.c

```diff
@@ -5040,11 +5040,23 @@ fu_engine_get_host_machine_id (FuEngine *self)
 static void
 fu_engine_add_security_attrs_tainted (FuEngine *self, FuSecurityAttrs *attrs)
 {
+	gboolean disabled_plugins = FALSE;
+	GPtrArray *blacklist = fu_config_get_blacklist_plugins (self->config);
 	FwupdSecurityAttr *attr = fwupd_security_attr_new ("org.fwupd.Hsi.Plugins");
 	fwupd_security_attr_set_name (attr, "fwupd plugins");
 	fwupd_security_attr_add_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE);
+	for (guint i = 0; i < blacklist->len; i++) {
+		const gchar *name_tmp = g_ptr_array_index (blacklist, i);
+		if (g_strcmp0 (name_tmp, "test") != 0 &&
+		    g_strcmp0 (name_tmp, "invalid") != 0) {
```

Comment:
Not sure about the hardcoding, but for these two it's probable fine.

Reply:
The other option is to only include these when plugin_dummy is compiled, but I think for distro purposes that's all the time, so I think I would rather keep them hardcoded.

Reply:
Ya, I think two is fine.